test/helper.rb

```diff
@@ -250,8 +250,10 @@ def md.directory?
       'pkginfo -l SUNWzfsr' => cmd.call('pkginfo-l-SUNWzfsr'),
       # solaris 11 package manager
       'pkg info system/file-system/zfs' => cmd.call('pkg-info-system-file-system-zfs'),
-      # dpkg-query package list
+      # dpkg-query all packages
       "dpkg-query -W -f='${db:Status-Abbrev} ${Package} ${Version}\\n'" => cmd.call('dpkg-query-W'),
+      # rpm query all packages
+      "rpm -qa --queryformat '%{NAME}   %{VERSION}-%{RELEASE}\\n'" => cmd.call('rpm-qa-queryformat'),
```

Comment:
I am okay with taking that approach right now. We should document it in code to make other developers aware of the intended use of the whitespace